com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/message/Messages.java

```diff
@@ -173,6 +173,7 @@
     public static final String PURGE_DELETE_REQUEST_SPACE_FROM_CONFIGURATION_TABLES = "All delete request spaces after date: {0} will be deleted from configuration tables.";
     public static final String RETRIEVED_USER_TOKEN = "Retrieved token for user: {0} with expiration time: {1}";
     public static final String FSS_CACHE_UPDATE_TIMEOUT = "Fss cache update timeout: {0} minutes";
+    public static final String THREAD_MONITOR_UPDATE_IN_MILIS = "Flowable thread monitor will  update every {0} miliseconds";

```

Comment:
Typo: Should be `THREAD_MONITOR_UPDATE_IN_MILLIS` and `milliseconds` in the string
Also, there's an extra space in the string.